numerics/чебышёв_series_body.hpp

```diff
@@ -28,6 +28,27 @@ template<typename Vector>
   two_over_duration_ = 2 / duration;
 }
 
+
+template<typename Vector>
+ЧебышёвSeries<Vector>::ЧебышёвSeries(ЧебышёвSeries&& other)
+    : coefficients_(std::move(other.coefficients_)),
+      degree_(other.degree_),
```

Comment:
Readability?  Why have a call to `std::move` when it has no usefulness whatsoever.